src/handlers.py

```diff
@@ -60,17 +60,19 @@ def all_contact(contacts:AddressBook) -> str:
     if not contacts.data:
```

Comment:
Here is what I got after testing your code with fake data

<img width="759" alt="image" src="https://github.com/LynxUA/yolo-address-book/assets/31358665/86807d11-35a2-487c-ba15-ffd9c159236f">

It would be nice to extend address block somehow and make the email section static in turms of the size.